### PR TITLE
OP-1049 Fix error from OP-999

### PR DESCRIPTION
**Pull request by nlazzer1**
See OP-1049

### CODE REVIEW

File: src/main/java/org/isf/stat/manager/JasperReportsManager.java [hunk 1]
```diff
@@ -959,10 +959,9 @@ private void addSubReportsBundleParameters(String jasperFileFolder, String jaspe
 		}
 	}
 
-	private void addReportBundleParameter(String jasperParameter, String jasperFileFolder, String jasperFileName, Map<String, Object> parameters) {
+	private void addReportBundleParameter(String jasperParameter, String jasperFileName, Map<String, Object> parameters) {
 		try {
-			ResourceBundle resourceBundle = ResourceBundle.getBundle(jasperFileFolder + File.separator + jasperFileName, new Locale(GeneralData.LANGUAGE),
-							new UTF8Control());
+			ResourceBundle resourceBundle = ResourceBundle.getBundle(jasperFileName, new Locale(GeneralData.LANGUAGE), new UTF8Control());
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/stat/manager/JasperReportsManager.java:964]: Can't `new Locale(GeneralData.LANGUAGE)` be a static class variable as you can't change the language without restarting OH.   That way it isn't reallocated for every report.   Just a thought ....

Comment (nlazzer1) [reply src/main/java/org/isf/stat/manager/JasperReportsManager.java:964]: Hi David, I restored that function to how it was before my last change, it was already like that. I'll move the creation to the beginning then as you suggest. Thank you very much

Comment (nlazzer1) [reply src/main/java/org/isf/stat/manager/JasperReportsManager.java:964]: The mainMenu set Locale.setDefault(new Locale(GeneralData.LANGUAGE)), so we can use the default locale

**Review by mwithi — COMMENTED** [review]
LGTM. Thanks a lot!

**Review by mwithi — APPROVED** [review]

File: src/main/java/org/isf/stat/manager/JasperReportsManager.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/stat/manager/JasperReportsManager.java]: is it ok now @dbmalkovsky? can we resolve the conversation?

**Review by dbmalkovsky — APPROVED** [review]
LGTM

File: src/main/java/org/isf/stat/manager/JasperReportsManager.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/stat/manager/JasperReportsManager.java]: yes

**Pull request merged**